<?xml version="1.0"?>
<launch>
	<!-- LAUNCH INTERFACE -->
	<arg name="hand_id" default="1"/>

	<!-- LAUNCH IMPLEMENTATION -->
	<param name="device_id" value="$(arg hand_id)"/>
	<node name="soft_hand_HW" pkg="aml_pisa_hand" type="pisa_soft_hand" respawn="false" output="screen"/>

</launch>